src/app/components/form/Select/index.tsx

```diff
@@ -5,7 +5,7 @@ type Props = React.SelectHTMLAttributes<HTMLSelectElement> & {
 function Select({ children, value, name, onChange }: Props) {
   return (
     <select
-      className="w-full h-14 border-gray-300 rounded-md shadow-sm text-gray-700 bg-white"
+      className="w-full h-14 border-gray-300 rounded-md shadow-sm text-gray-700 bg-white dark:bg-gray-800 dark:text-gray-200 dark:border-gray-500"
```

Comment:
```suggestion
      className="w-full border-gray-300 rounded-md shadow-sm text-gray-700 bg-white dark:bg-gray-800 dark:text-gray-200 dark:border-gray-500"
```

Reply:
Nice! Could you also remove the "h-14" so it matches height of our buttons?

Reply:
Yep, I also thought the button looked a bit big.